api/api_gateway.go

```diff
@@ -35,15 +35,11 @@ import (
 //  * Generate openrpc blobs
 
 type Gateway interface {
-	MpoolPending(context.Context, types.TipSetKey) ([]*types.SignedMessage, error)
```

Comment:
changes in here are just reorgs to get them into their groups and close to their neighbours, I've not been thorough, just cleaning up the obvious bits out of place